docs/use-cases.md

```
@@ -0,0 +1,1077 @@
+
+pgatk generate-decoy \
+    --input novel_orf_target.fa \
+    --output novel_orf_target_decoy.fa \
+    --method decoypyrat
+```
+
+!!! tip
+    After searching, use `digest-mutant-protein` to extract only the novel
+    peptides that do not appear in the canonical proteome:
+
+    ```bash
```

markdownlint-cli2 (0.21.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)